libeos-updater-util/extensions.c

```diff
@@ -414,6 +415,22 @@ get_summary (EosExtensions *extensions,
         return FALSE;
       }
 
+  g_clear_error (&local_error);
+  summary_info = g_file_query_info (summary,
+                                    G_FILE_ATTRIBUTE_TIME_MODIFIED,
+                                    G_FILE_QUERY_INFO_NONE,
+                                    cancellable,
+                                    &local_error);
+  if (summary_info == NULL)
+    {
+      if (!g_error_matches (local_error, G_IO_ERROR, G_IO_ERROR_NOT_FOUND))
+        {
+          g_propagate_error (error, g_steal_pointer (&local_error));
+          return FALSE;
+        }
+    }
+  extensions->summary_modification_time_secs = g_file_info_get_attribute_uint64 (summary_info,
```

Comment:
It’s possible for `summary_info` to be `NULL` at this point, if the error was `G_IO_ERROR_NOT_FOUND` above.